### ISSUE TITLE
Open testing error 2/3 - Join and split group

### ISSUE THREAD

**Issue by Bob-Murphy** [official-antistasi-community/A3-Antistasi#537]
**### Please assign yourself to the issue or comment your name when you start working on it.**

There errors happened together:

```
12:56:13 "Splitting the crew of B Bravo 3-1:1, type of B_T_Quadbike_01_F"
12:56:13 "Crew is [[B Bravo 3-1:1,""driver"",-1,[],false]]"
12:56:13 "Splitting the crew of 171eaf56040# 2270279: quadbike_01_f.p3d, type of B_T_Quadbike_01_F"
12:56:13 "Crew is []"
12:56:13 Error in expression <_group;
} forEach _crew;

[_crewGroups, _groupName];
>
12:56:13   Error position: <_groupName];
>
12:56:13   Error Undefined variable in expression: _groupname
12:56:13 File functions\Base\fn_splitVehicleCrewIntoOwnGroups.sqf [A3A_fnc_splitVehicleCrewIntoOwnGroups]..., line 32
12:56:13 "Splitting the crew of B Alpha 3-6:1, type of B_T_Quadbike_01_F"
12:56:13 "Crew is [[B Alpha 3-6:1,""driver"",-1,[],false]]"
12:56:13 "Splitting the crew of 171eaea8080# 2270343: quadbike_01_f.p3d, type of B_T_Quadbike_01_F"
12:56:13 "Crew is []"
12:56:13 Error in expression <_group;
} forEach _crew;

[_crewGroups, _groupName];
>
12:56:13   Error position: <_groupName];
>
12:56:13   Error Undefined variable in expression: _groupname
12:56:13 File functions\Base\fn_splitVehicleCrewIntoOwnGroups.sqf [A3A_fnc_splitVehicleCrewIntoOwnGroups]..., line 32
12:56:13 "Splitting the crew of B Bravo 2-4:1, type of B_T_Quadbike_01_F"
12:56:13 "Crew is [[B Bravo 2-4:1,""driver"",-1,[],false]]"
12:56:13 Error in expression <Each _groups;

_joinedGroup setGroupId [_groupName];

_joinedGroup;>
12:56:13   Error position: <_groupName];

_joinedGroup;>
12:56:13   Error Undefined variable in expression: _groupname
12:56:13 File functions\Base\fn_joinMultipleGroups.sqf [A3A_fnc_joinMultipleGroups]..., line 22
12:56:13 Error in expression <Each _groups;

_joinedGroup setGroupId [_groupName];

_joinedGroup;>
12:56:13   Error position: <_groupName];

_joinedGroup;>
12:56:13   Error Undefined variable in expression: _groupname
12:56:13 File functions\Base\fn_joinMultipleGroups.sqf [A3A_fnc_joinMultipleGroups]..., line 22
```

**Comment by jaj22** [comment]
There's a separate report of quadbike crew dying immediately on spawn, which would explain why the crew list is already empty when it reaches splitVehicleCrew.

I'll be looking into it.

**Comment by jaj22** [comment]
Could do with the full RPT from this one, as the convoy code should spam out a lot of interesting info.

The method of spacing out the convoy spawns is suspect:
https://github.com/official-antistasi-community/A3-Antistasi/blob/038069f269a3bba266bb7776a52abd36f6c03ba1/A3-Antistasi/functions/Convoy/fn_spawnConvoy.sqf#L95

That's probably not supposed to be 108km/h. This doesn't appear to directly kill quad bikes, but they bleed velocity quickly, so a heavier vehicle spawned afterwards would probably smash them. The vehicle crew aren't given any orders until the whole convoy has been spawned.

**Comment by Spoffy** [comment]
Fixed in f37bc4bd14e9d8e8520df034801213a66a1c39c2
